spec/index.bs

```diff
@@ -4962,54 +4951,59 @@ dictionary GPUExternalTextureDescriptor : GPUObjectDescriptorBase {
 
                 1. Let |source| be |descriptor|.{{GPUExternalTextureDescriptor/source}}.
 
-                1. If |source| <l spec=html>[=is not origin-clean=]</l>,
-                    throw a {{SecurityError}} and stop.
+                1. If the current image contents of |source| are the same as the most recent
+                    {{GPUDevice/importExternalTexture()}} call with an identical |descriptor|,
+                    returning |previousResult|, and the user agent chooses to reuse it:
```

Comment:
It took me several readings of this algorithm to realize that this was actually declaring and setting `|previousResult|`. I think we could stand to make that clearer.